Sources_v3/StreamChatUI/ChatChannelList/ChatChannelAvatarView.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+import StreamChat
+
+open class ChatChannelAvatarView<ExtraData: UIExtraDataTypes>: AvatarView {
+    
+    public var channel: _ChatChannel<ExtraData>? {
+        didSet { updateContent() }
+    }
+    
+    override open func setupLayout() {
+        widthAnchor.constraint(equalTo: heightAnchor, multiplier: 1).isActive = true
+        embed(imageView, insets: .init(top: 8, leading: 8, bottom: -8, trailing: -8))
+    }
+    
+    open func updateContent() {
+        guard let channel = channel else { return }
```

Comment:
If `channel` doesn't exist I'd set `imageView.image = nil` other than preserving the previous one